pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1782,3 +1804,37 @@ func getServingCertsMode(remoteKubernetes *ispnutil.Kubernetes) string {
 	}
 	return ""
 }
+
+func (r *ReconcileInfinispan) applyLabelsToCoordinatorsPod(podList *corev1.PodList, infinispan *infinispanv1.Infinispan, cluster ispnutil.ClusterInterface, logger logr.Logger) bool {
+	pass, err := cluster.GetPassword(consts.DefaultOperatorUser, infinispan.GetSecretName(), infinispan.GetNamespace())
+	coordinatorFound := false
+	if err != nil {
+		return coordinatorFound
+	}
+	protocol := string(infinispan.GetEndpointScheme())
+	for _, item := range podList.Items {
+		cacheManagerInfo, err := cluster.GetCacheManagerInfo(consts.DefaultOperatorUser, pass, "default", item.Name, item.Namespace, protocol)
+		if err == nil {
+			lab, ok := item.Labels["coordinator"]
+			if cacheManagerInfo["coordinator"].(bool) {
+				if !ok || lab != "true" {
+					item.Labels["coordinator"] = "true"
+					err = r.client.Update(context.TODO(), &item)
+				}
+				coordinatorFound = (err == nil)
```

Comment:
Feedback from GoLand IDE: Redundant parentheses

Reply:
Yeah, I would keep them just for human readability